var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -121,6 +122,10 @@ def setup_run_environment(self, env):
             # dynamicaly load the python bindings library
             env.prepend_path("LD_LIBRARY_PATH", self.spec["sio"].libs.directories[0])
 
+        if self.satisfies("@0.16.1:"):
+            # Frame header needs to be available for python bindings
+            env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
```

Comment:
This needs to be fixed because only self.spec has .satisfies:
```suggestion
        if self.spec.satisfies("@0.16.1:"):
            # Frame header needs to be available for python bindings
            env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
```

Reply:
thanks :)